app/[locale]/faculty-and-staff/page.tsx

```diff
@@ -1,9 +1,157 @@
+      name: 'Arun Goel',
+      designation: 'Professor (Head of the Department)',
+      email: 'drarun_goel@yahoo.co.in',
+      phone: '01744-233349, 01744-233300',
+      id: 7,
+    },
+    {
+      profileImage:
+        'https://s3-alpha-sig.figma.com/img/6698/6d20/c7aa47a52ea27a318d8cb53424a5e808?Expires=1711324800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=WNeY~2a8t85zJ-C5kvhI5xs6pVrmFYQg~9z367Cqzlpy9cn6fLBEp9IoEpsvC4wclpaWw9BZ0azdAa~t2HE0-B9gkfqGS4ABg69JE9whHNqFslgrk6yCDMG21zt9YcQShVd7OwYDLYBDAy7ChO2MWyhBfowI93P4sLxKhN~B~m44wrmCdxPbgnM60qHx8O02tfWa6L79w~FRc4FF-AzoypfBMaVveEPLFEhqfKrOjF~K3G8By3yx0-0WP1VNoHgTX06Jqw1Uf7LdB76OzGblx~2Dyn4UB9zW1bQxpBiKb8Fefk5fSStS3p2wZuKpk4dfUKkP0locrEfrt7rTEKhFFg__',
+      name: 'Arun Goel',
+      designation: 'Professor (Head of the Department)',
+      email: 'drarun_goel@yahoo.co.in',
+      phone: '01744-233349, 01744-233300',
+      id: 8,
+    },
+    {
+      profileImage:
+        'https://s3-alpha-sig.figma.com/img/6698/6d20/c7aa47a52ea27a318d8cb53424a5e808?Expires=1711324800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=WNeY~2a8t85zJ-C5kvhI5xs6pVrmFYQg~9z367Cqzlpy9cn6fLBEp9IoEpsvC4wclpaWw9BZ0azdAa~t2HE0-B9gkfqGS4ABg69JE9whHNqFslgrk6yCDMG21zt9YcQShVd7OwYDLYBDAy7ChO2MWyhBfowI93P4sLxKhN~B~m44wrmCdxPbgnM60qHx8O02tfWa6L79w~FRc4FF-AzoypfBMaVveEPLFEhqfKrOjF~K3G8By3yx0-0WP1VNoHgTX06Jqw1Uf7LdB76OzGblx~2Dyn4UB9zW1bQxpBiKb8Fefk5fSStS3p2wZuKpk4dfUKkP0locrEfrt7rTEKhFFg__',
+      name: 'Arun Goel',
+      designation: 'Professor (Head of the Department)',
+      email: 'drarun_goel@yahoo.co.in',
+      phone: '01744-233349, 01744-233300',
+      id: 9,
+    },
+  ]);
+  const allDepartments = [
+    'Computer Engineering',
+    'Computer Application',
+    'Chemistry',
+    'Civil Engineering',
+    'Electrical Engineering',
+    'Electronical and Communication Engineering',
+    'School of Renewable Energy and Efficiency',
+    'Mathematics',
+    'Mechanical Engineering',
+    'Physics',
+    'School of VLSI Design and Embedded System',
+  ];
+  return (
+    <div className="container my-16 flex gap-10">
+      {/* BREADCRUMB + ALL DEPARTMENTS LIST */}
+      <div className="flex flex-col items-start pt-16">
+        {/* All Departments */}
+        <div className="sticky top-[110px] flex w-[350px] flex-col items-start justify-center gap-9 rounded-xl border border-neutral-400 bg-shade-light p-5 2xl:w-[480px]">
+          <p className="font-sans text-xl font-bold text-primary-700">
+            All Departments
+          </p>
+          {allDepartments.map((department) => (
+            <p
+              className="font-sans text-lg font-bold text-shade-dark"
+              key={department}
+            >
+              {department}
+            </p>
```

Comment:
These departments are supposed to be clickable which then sort the faculty based on their department category.